examples/sol-anchor-contract/scripts/invoke.ts

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const assert = require("assert");
+const anchor = require("@project-serum/anchor");
+
+let ethToUSD = "EdVCmQ9FSPcVe5YySXDPCRmc8aDQLKJ9xvYBMZPie1Vw";
+let usdtToUSD = "38xoQ4oeJCBrcVvca2cGk7iV1dAfrmTR1kmhSCJQ8Jto";
+
+const provider = anchor.AnchorProvider.env();
+anchor.setProvider(provider);
+
+const config = anchor.web3.Keypair.generate();
+const program = anchor.workspace.ExampleSolAnchorContract;
+let programId = program.programId;
+var programKey;
+try {
+    let data = fs.readFileSync(
+        'target/deploy/example_sol_anchor_contract-keypair.json'
+    );
+    programKey = anchor.web3.Keypair.fromSecretKey(
+        new Uint8Array(JSON.parse(data))
+    );
+} catch (error) {
+    throw new Error("Please run `anchor build` first.");
+}
+
+try {
+    assert(programId.equals(programKey.publicKey));
+} catch (error) {
```

Comment:
I know where you're trying to go with this check but with the improvements in the deployment process that I suggested it now longer works.